examples/run_examples.sh

```diff
@@ -251,6 +251,14 @@ if [ $WOLFCRYPT_ENABLE -eq 1 ]; then
     ./examples/keygen/keyload ecckeyblobeh.bin -ecc -eh >> $TPMPWD/run.out 2>&1
     RESULT=$?
     [ $RESULT -ne 0 ] && echo -e "keyload endorsement ecc failed! $RESULT" && exit 1
+
+    # Test KeyGen with custom password
+    ./examples/keygen/keygen rsakeyblobeh.bin -rsa -eh -t -auth=custompass >> $TPMPWD/run.out 2>&1
+    RESULT=$?
+    [ $RESULT -ne 0 ] && echo -e "keygen endorsement rsa failed! $RESULT" && exit 1
+    ./examples/keygen/keygen rsakeyblobeh.bin -rsa -eh -auth=custompass >> $TPMPWD/run.out 2>&1
```

Comment:
Looks like the password isn't needed to load the key and get a handle. It is only needed when you use it later to sign with it. So you can also add tests like: `./examples/keygen/keyload rsakeyblobeh.bin -eh`

Reply:
Honestly not much value in adding this test until it actually uses it. Please add a TODO note.